lib/MetaCPAN/Web/Model/API/Release.pm

```diff
@@ -4,6 +4,9 @@ use namespace::autoclean;
 
 extends 'MetaCPAN::Web::Model::API';
 
+use List::Util qw(first);
+use List::MoreUtils qw(uniq);
```

Comment:
I think we can change this to `use List::AllUtils qw( first uniq );`  If `List::AllUtils` is not in our deps, we should add it.